oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/cluster/ClusterModule.java

```diff
@@ -32,7 +32,9 @@ public ClusterModule() {
     public Class[] services() {
         return new Class[] {
             ClusterRegister.class,
-            ClusterNodesQuery.class
+            ClusterNodesQuery.class,
+            ClusterWatcherRegister.class,
+            ClusterCoordinator.class
```

Comment:
As `ClusterCoordinator` inherits `ClusterWatcherRegister`, do we need to expose both?